packages/opentelemetry-plugin-postgres/opentelemetry-plugin-pg-pool/src/pg-pool.ts

```diff
@@ -13,3 +13,123 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+import { BasePlugin } from '@opentelemetry/core';
+import { CanonicalCode, SpanKind } from '@opentelemetry/types';
+import { AttributeNames } from './enums';
+import * as shimmer from 'shimmer';
+import * as pgPoolTypes from 'pg-pool';
+import {
+  PostgresPoolPluginOptions,
+  PgPoolCallback,
+  PgPoolExtended,
+} from './types';
+import * as utils from './utils';
+
+export class PostgresPoolPlugin extends BasePlugin<typeof pgPoolTypes> {
+  protected _config: PostgresPoolPluginOptions;
+
+  static readonly COMPONENT = 'pg-pool';
+  static readonly DB_TYPE = 'sql';
+
+  readonly supportedVersions = ['^2.0.7'];
+
+  constructor(readonly moduleName: string) {
+    super();
+    this._config = {};
+  }
+
+  protected patch(): typeof pgPoolTypes {
+    if (this._moduleExports.prototype.connect) {
```

Comment:
Is this if statement necessary? Is there any case where `connect` would be missing?